quiet.js

```diff
@@ -178,32 +193,47 @@ var Quiet = (function() {
      * @function init
      * @memberof Quiet
      * @param {object} opts - configuration options
-     * @param {string} opts.profilesPrefix - path prefix to quiet-profiles.json
+     * @param {string} [opts.profilesPath] - path to quiet-profiles.json
      *   this file configures transmitter and receiver parameters
-     * @param {string} opts.memoryInitializerPrefix - path prefix to quiet-emscripten.js.mem
-     * @param {string} [opts.libfecPrefix] - path prefix to libfec.js
+     *   defaults to "quiet-profiles.json"
+     * @param {string} [opts.profilesPrefix] [deprecated] - prefix of path to
+     *   quiet-profiles.json. Use opts.profilesPath instead.
+     * @param {string} [opts.emscriptenPath] - path to quiet-emscripten.js
+     *   defaults to "quiet-emscripten.js"
+     * @param {string} opts.memoryInitializerPath - path to quiet-emscripten.js.mem
+     *   defaults to "quiet-emscripten.js.mem"
+     * @param {string} opts.memoryInitializerPrefix - prefix of path to quiet-emscripten.js.mem
+     *   Use opts.memoryInitializerPath instead
      * @param {function} [opts.onReady] - Quiet ready callback
      * @param {onError} [opts.onError] - User errback function
      * @example
      * Quiet.init({
-     *   profilesPrefix: "/",  // fetches /quiet-profiles.json
-     *   memoryInitializerPrefix: "/",  // fetches /quiet-emscripten.js.mem
-     *   libfecPrefix: "/",  // fetches /libfec.js
```

Comment:
n.b. we should be able to get rid of the fec jankiness now. The reason I didn't bundle it before was because it was LGPL, and I was afraid of license taint. Now fec functionality is provided by Quiet's own libcorrect which is BSD-3 licensed, so it's just bundled in (or should be by this new build).